SevenSlots/SevenSlots/SevenSlots/View/LoginErrorPopup.xaml.cs

```diff
@@ -10,9 +10,9 @@
 namespace SevenSlots.View
 {
     [XamlCompilation(XamlCompilationOptions.Compile)]
-    public partial class OtherPage : ContentPage
+    public partial class LoginErrorPopup : Rg.Plugins.Popup.Pages.PopupPage
```

Comment:
Maybe instead of using the whole  Rg.Plugins.Popup.Pages.PopupPage you could include it in your usings.

Reply:
It may look better, but it imports unnecessary stuff that way

Reply:
You can import it without adding unnecessary stuff

Reply:
Just like in python